Sources/Components/IconView/IconSize.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+extension Warp {
+    /// Icon sizes that can be used in `IconView`.
+    ///
+    /// `IconSize` defines three standard sizes: `small` (16x16), `default` (24x24), `large` (32x32),
+    /// and a `custom` size that can be specified by the user.
+    public enum IconSize: CaseIterable, Hashable {
+        public static var allCases = [Warp.IconSize.small, .default, .large]
```

Comment:
This property cam be omitted, since it will be generated by compiler.

Reply:
allCases? No it won't. It has a custom one also :D which has a associated value.

Reply:
Then it's not really `CaseIterable`, right? =D

Reply:
That's on Apple not us 😅 That custom case is not part of Warp really, it is just there to make sure that users can override it if needed.

Maybe in this case it makes sense to instead of having it confirm to `CaseIterable` just have it on the demo app, since `allCases` is only for demoing purposes.